Update high-throughput-order-processing.md (#29)
AdobeDocs/commerce-operations.en: help/performance/high-throughput-order-processing.md

**Enable Inventory Check On Cart Load** is enabled (set to Yes) by default. To disable the inventory check when loading the cart, set **[!UICONTROL Enable Inventory Check On Cart Load]** to `No` in the Admin UI **Stores** > **Configuration** > **Catalog** > **Inventory** > **Stock Options** section. See [Configure Global Options][global] and [Catalog Inventory][inventory] in the _User Guide_.

> ## Slave connections

Reviewer: It may be more appropriate and enticing to use a different title. What about "Load balancing" ?

```suggestion
## Load balancing
```

> Enabling SLAVE connections for the MySQL database and Redis can help to spread out the load across different nodes.

Reviewer: ```suggestion
You can help balance the load across different nodes by enabling secondary connections for the MySQL database and Redis instance.
```

> Adobe Commerce can read multiple databases or Redis asynchronously. Updating the .magento.env.yaml file by setting to true the values MYSQL_USE_SLAVE_CONNECTION and REDIS_USE_SLAVE_CONNECTION to use a read-only connection to the database to receive read-only traffic on a non-master node. This improves performance through load balancing because only one node needs to handle read-write traffic. Set to false to remove any existing read-only connection array from the env.php file.

Reviewer: We have to be explicit about the use of the cloud infrastructure since this guide was initially  written for Commerce on-premises. I changed the wording to accommodate the use of load balancing and use a different tone. I also added links to the variable definitions in the Cloud guide, for further research.

```suggestion
Adobe Commerce can read multiple databases or Redis instances asynchronously. If you are using Commerce on cloud infrastructure, you can configure the secondary connections by editing the [MYSQL_USE_SLAVE_CONNECTION](https://devdocs.magento.com/cloud/env/variables-deploy.html#mysql_use_slave_connection) and [REDIS_USE_SLAVE_CONNECTION](https://devdocs.magento.com/cloud/env/variables-deploy.html#redis_use_slave_connection) values in the `.magento.env.yaml` file. Only one node needs to handle read-write traffic, so setting the variables to `true` results in creating a secondary connection for read-only traffic. Set the values to `false` to remove any existing read-only connection array from the `env.php` file.
```

> Example of how  `magento.env.yaml` should looks like

Reviewer: ```suggestion
Example of the `.magento.env.yaml` file:
```
